src/api/status/env.local

```diff
@@ -1 +1,2 @@
 PLANET_PORT=1111
+API_URL=http://localhost/v1/posts
```

Comment:
This is not necessary, as mentioned [here](https://github.com/Seneca-CDOT/telescope/pull/2471#discussion_r749590703)

Reply:
Actually we had to add cors to posts api and run it locally so that we could retrieve x-total-count from the header. That's why he put `localhost/v1/posts` there.

Reply:
but it shouldn't be included in the commit right? And for cors, it is passed inside the Satellite as an option, not a separate express app. 
https://github.com/Seneca-CDOT/telescope/blob/1407a343df43b0c2d2bac941e4351af1ced68a43/src/api/posts/src/index.js#L7-L10

Reply:
Yeah it shouldn't be in `env.local`